Locales/enUS.lua

```diff
@@ -131,6 +131,12 @@ AUCTIONATOR_LOCALES.enUS = function()
   L["SHOPPING_TAB_HEADER"] = "Auctionator - Shopping Lists"
   L["INFO_TAB_HEADER"] = "Auctionator - Info"
 
+  L["SELLING_TAB"] = "Sell"
+  L["SELLING_TAB_HEADER"] = "Auctionator - Sell"
```

Comment:
Shouldn't the tab be named "Selling" - to differentiate it from the blizzard "Sell"?